libs/net/services/Managers/IngestManager`.cs

```diff
@@ -83,6 +88,11 @@ public override async Task RunAsync()
                     var delayMS = ingest.IngestSchedules.Where(s => s.Schedule?.DelayMS > 0).Min(s => s.Schedule?.DelayMS) ?? delay;
                     delay = delayMS < delay ? delayMS : delay;
 
+                    // Fetch the latest version of the ingest and check if the location is still valid or not.
+                    var theLatest = await this.Api.GetIngestAsync(ingest.Id);
+                    var isExpected = theLatest?.DataLocations.Any(d => d.Name.ToLower() == this.Options.DataLocation.ToLower());
+                    if (!isExpected.HasValue || !isExpected.Value) continue;
```

Comment:
Easier to read if you just go `isExpected == true`.  If the result is false does this stop the process if it is already running?

Reply:
Updated, thanks